Allow to install headers needed by the headers of the interfaces separately
from the headers needed for server. We need this to allow building PostgreSQL
server with LDAP support, but libpq without dependency on LDAP.

--- a/src/include/Makefile
+++ b/src/include/Makefile
@@ -27,7 +27,8 @@
 	port/win32/sys portability
 
 # Install all headers
-install: all installdirs
+install: install-interfaces install-server
+install-interfaces: all installdirs
 # These headers are needed by the public headers of the interfaces.
 	$(INSTALL_DATA) $(srcdir)/postgres_ext.h   '$(DESTDIR)$(includedir)'
 	$(INSTALL_DATA) $(srcdir)/libpq/libpq-fs.h '$(DESTDIR)$(includedir)/libpq'
@@ -40,6 +41,7 @@
 	$(INSTALL_DATA) $(srcdir)/port.h         '$(DESTDIR)$(includedir_internal)'
 	$(INSTALL_DATA) $(srcdir)/postgres_fe.h  '$(DESTDIR)$(includedir_internal)'
 	$(INSTALL_DATA) $(srcdir)/libpq/pqcomm.h '$(DESTDIR)$(includedir_internal)/libpq'
+install-server: all installdirs
 # These headers are needed for server-side development
 	$(INSTALL_DATA) pg_config.h     '$(DESTDIR)$(includedir_server)'
 	$(INSTALL_DATA) pg_config_ext.h '$(DESTDIR)$(includedir_server)'
